cloudinit/sources/DataSourceLXD.py

```diff
@@ -211,13 +203,19 @@ def network_config(self) -> dict:
         If none is present, then we generate fallback configuration.
         """
         if self._network_config == sources.UNSET:
```

Comment:
Here, we are hardening the computation of `self.network_config` to work even if `self._get_data` wasn't called previously.